annotator-core/src/main/java/edu/ucr/cs/riple/core/Annotator.java

```diff
@@ -157,7 +157,7 @@ private void executeNextIteration(
     injector.injectFixes(selectedFixes);
     // Update log.
     context.log.updateInjectedAnnotations(
-        selectedFixes.stream().map(fix -> fix.change).collect(Collectors.toSet()));
+        selectedFixes.stream().flatMap(fix -> fix.changes.stream()).collect(Collectors.toSet()));
```

Comment:
For after this PR: it'd be good to add Google Java Format formatting to this repo, enforced via Spotless.  It will get rid of long lines like this one that are hard to read.

Reply:
Sure, will do #248